source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "dom-chef";
+import elementReady from "element-ready";
+import select from "select-dom";
+import * as pageDetect from "github-url-detection";
+import LinkExternalIcon from "octicon/link-external.svg";
+
+import features from ".";
+
+function getCurrentUsername(): string {
+	return location.pathname.replace(/(^\/)/g, "").split("/")[0];
+}
+
+function getGithubIoLink(username: string): string {
+	return `${username}.github.io`;
+}
+
+async function repoTabInit(): Promise<void> {
+	const username = getCurrentUsername();
+	const githubIoLink = getGithubIoLink(username);
+	const repo = select(`a.rgh-discussion-links[href$="${githubIoLink}"]`);
+	repo?.parentElement?.appendChild(
+		<a
+			href={`https://${githubIoLink}`}
+			target="black"
+			rel="noopener noreferrer"
+		>
+			<LinkExternalIcon />
+		</a>
+	);
+	return;
+}
+
+async function githubIoRepoInit(): Promise<void> {
+	const repositoryName = pageDetect.utils.getRepoURL().split("/")[1];
+	const githubIoLink = getGithubIoLink(getCurrentUsername());
+	const link = (await elementReady(".BorderGrid-cell .octicon-link"))!;
+	if (repositoryName.endsWith(".github.io")) {
+		if (!link) {
+			const aboutTab = select(".BorderGrid-cell");
+			aboutTab?.appendChild(
```

Comment:
Only use `?.` if there's an unavoidable situation where it might not exist. `?.` silences errors so we don't want that. **This applies to every where you used `?.`**

```suggestion
			select(".BorderGrid-cell")!.appendChild(
```